tests/test_template.py

```
@@ -1,102 +1,450 @@
         project_name="CompileExample",
         package_name="compile_example",
-        license_year=datetime.now(tz=UTC).year,
-        license_holder="Compile Dev",
-        license_email="compile@example.com",
         flavour=flavour,
     )
     project.run("cargo check --all-targets --all-features")
 
+
+@pytest.mark.parametrize("flavour", [LIB, APP])
+def test_generated_tooling_contracts(
```

Ruff (0.15.13)

[warning] Too many statements (75 > 50)

(PLR0915)